.gitignore

```diff
@@ -4,8 +4,8 @@
 *.pyc
 __pycache__
 db.sqlite3
-migrations
```

Comment:
Why are we taking this out?

Reply:
It was replaced with some more precise ignore-y stuff at the bottom of the file which keeps the init files intact. I think this should be how main is already??